embeddings/bq-vector-search-log-outlier-detection.ipynb

```diff
@@ -0,0 +1,1474 @@
+{
```

Comment:
<p>Can you add links for Vertex AI and BigQuery.</p><p> </p><p>Does textembedding-gecko count as an LLM?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/381/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>Added. </p><p>To my understanding textembedding-gecko is a LLM as it's part of the PaLM 2 family of LLMs. That said, I'm changing it to "Generative AI model" to be consistent with our own docs: <a href="https://cloud.google.com/vertex-ai/docs/generative-ai/learn/overview#genai-models" target="_blank">https://cloud.google.com/vertex-ai/docs/generative-ai/learn/overview#genai-models</a></p>

Reply:
<p>You can directly link the &nbsp;<code><a href="https://cloud.google.com/vertex-ai/docs/generative-ai/embeddings/get-text-embeddings" target="_blank">textembedding-gecko</a></code> docs here: <a href="https://cloud.google.com/vertex-ai/docs/generative-ai/embeddings/get-text-embeddings" target="_blank">https://cloud.google.com/vertex-ai/docs/generative-ai/embeddings/get-text-embeddings</a></p>

Reply:
Done. linked to docs.